packages/mc-scripts/package.json

```diff
@@ -40,13 +40,13 @@
     "@commercetools-frontend/mc-html-template": "13.10.0",
     "@svgr/webpack": "4.3.0",
     "babel-loader": "8.0.6",
-    "babel-plugin-react-intl": "3.2.1",
-    "browserslist": "4.6.2",
+    "babel-plugin-react-intl": "3.3.3",
+    "browserslist": "4.6.3",
     "clean-webpack-plugin": "1.0.1",
     "core-js": "3.1.4",
     "css-loader": "2.1.1",
     "cssnano": "4.1.10",
-    "file-loader": "3.0.1",
+    "file-loader": "4.0.0",
```

Comment:
minimum required nodejs version is 8.9.0

Reply:
Should be fine under development I think.